src/main/java/net/minestom/server/utils/location/RelativeVec.java

```diff
@@ -27,6 +27,10 @@ public RelativeVec(@NotNull Vec vec, @NotNull CoordinateType coordinateType, boo
         this.relativeZ = relativeZ;
     }
 
+    public @NotNull CoordinateType getCoordinateType() {
```

Comment:
`coordinateType` would be better, in case we make it a record

Reply:
:+1: fixed